### ISSUE TITLE
Tree-View in PR View

### ISSUE THREAD

**Issue by codeshaw** [isaacs/github#603]
When looking through larger PRs the file ordering of the PR becomes seemingly random, making it hard to follow the thread of other people's code before merging.

Is there any way to display the changes in a tree-view so that PRs can be navigated, instead of having to scroll back and forward between related code?

**Comment by cirosantilli** [comment]
Do you mean show a directory tree linking to the changed files?

**Comment by codeshaw** [comment]
Yes. This would allow the reviewer to find a "start point" that makes sense to them and to work through the changed code in a manner that makes sense logically as the location of the files within the tree imparts information as to what that file is for.

At the moment the PR review dumps all the files in a list, which makes it harder to grasp the deeper meaning of for larger, more complex changes.

**Comment by chvarunkumar1994** [comment]
As a workaround for Chrome users, [this](https://chrome.google.com/webstore/detail/octotree/bkhaagjahfmjljalopjnoealnfndnagc/related) extension works like charm. 

Having said that, May I know if there any thoughts on improving PR views, like suggested above?
